README.md

```diff
@@ -43,40 +46,29 @@ spec:
                 port:
                   number: 80
 ```
-
-<br>
-
-### How to configure the different options:
-1. [tls passthrough (podinfo would terminate tls)] Create a new virtual service, gateway for podinfo as a passthrough, and use an existing load balancer:
+Example of a service:
 ```yaml
-  annotations:
-    pepr.dev/gateway: podinfo
-    pepr.dev/ingress-selector: '{ "istio": "ingressgateway"}'
+apiVersion: v1
+kind: Service
+metadata:
+  # metadata.name will be the hostname
+  # TBD: how to configure the domain (default in the istio.ts)
+  name: myapp
+  namespace: myapp
+  labels:
+    pepr.dev/ingress: true
 
+spec:
+  selector:
+    app: myapp
+  ports:
+    - protocol: TCP
+      # TBD: how to configure which port to use, currently will always pick up 80/443
```

Comment:
do you want to make an issue to track this?

Reply:
if we decide to use this capability, we'll do that work, right now this is just finishing a dash days event and a way to test the integration with pepr 0.14.0 and the KFC.